pype/plugins/nuke/publish/collect_review.py

```diff
@@ -9,7 +9,7 @@ class CollectReview(pyblish.api.InstancePlugin):
     order = pyblish.api.CollectorOrder + 0.3
     label = "Collect Review"
     hosts = ["nuke"]
-    families = ["render", "render.local", "render.farm"]
+    families = ["write", "prerender"]
```

Comment:
You should only need `write` here as that is on both rendes and prerenders, Or alternatively use `render` and `prerender` to target the reviews just to those two.